tests/queries_test.go

```diff
@@ -1,15 +1,23 @@
-// +build system
-
```

Comment:
One man's opinion but I like this instead of the test short stuff

Reply:
Yes, it's better.  I was having random trouble and commented this out so I'll put it back.